ios/manifest

```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 3.2.0
+version: 4.0.0
 apiversion: 2
```

Comment:
do we need to update the apiversion too?

Reply:
I dont think so. I generated a new module for iOS and it was still set to 2